src/views/projects/Cob/CrudErrorModal.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
```

Comment:
Can we call this just `ErrorModal` or `CobErrorModal`? As CRUD stands for Create, Read, Update and Delete. We don't really have errors for read and delete.

Reply:
3af3c74